src/features/classification/flowTypes.js

```diff
@@ -44,11 +44,16 @@ type SharedLinkRestrictions = {
     accessLevel: SharedLinkAccessLevel,
 };
 
+type watermarkApplied = {
```

Comment:
Would it be preferable to name this consistently like `WatermarkRestriction`, or is there a reason why a different convention was used in this case?

Reply:
Was discussing with Yuvnesh on this, watermarking is not considered as a restriction like the other controls, though we have it as part of access policy, which is also why we are printing it with an extra bullet point in the short version.

Reply:
Copy in admin console: 
![Screen Shot 2022-01-20 at 3 08 36 PM](https://user-images.githubusercontent.com/20669641/150436591-c9e5162f-a174-42fb-a456-7034c0af9391.png)